contracts/nodes/VVVNodes.sol

```diff
@@ -113,11 +139,68 @@ contract VVVNodes is ERC721, ERC721URIStorage {
         if (!success) revert TransferFailed();
     }
 
+    ///@notice Sets the node activation threshold in staked $VVV
+    function setActivationThreshold(uint256 _activationThreshold) external onlyAuthorized {
+        if (_activationThreshold > activationThreshold) {
+            //update claimable balanaces of nodes which will become inactive as a result of the threshold increase
+            for (uint256 i = 1; i <= tokenId; i++) {
+                TokenData storage token = tokenData[i];
+                if (
+                    _isNodeActive(token.stakedAmount, activationThreshold) &&
+                    !_isNodeActive(token.stakedAmount, _activationThreshold)
+                ) {
+                    _updateClaimableFromVesting(token);
+                }
+            }
+        } else if (_activationThreshold < activationThreshold) {
+            //set vestingSince for nodes which will become active as a result of the threshold decrease
+            for (uint256 i = 1; i <= tokenId; i++) {
+                TokenData storage token = tokenData[i];
+                if (
+                    _isNodeActive(token.stakedAmount, _activationThreshold) &&
+                    !_isNodeActive(token.stakedAmount, activationThreshold)
+                ) {
+                    token.vestingSince = block.timestamp;
+                }
+            }
+        } else {
+            revert UnchangedActivationThreshold();
```

Comment:
The initial thought was to avoid wasting gas by looping through things, but that would never have been an issue since neither loop would have been entered - worth removing given that